lib/phoenix_profiler/live_view_listener.ex

```diff
@@ -134,4 +149,8 @@ defmodule PhoenixProfiler.LiveViewListener do
 
     send(pid, {:navigation, view})
   end
+
+  defp notify_event_duration(pid, duration) do
+    send(pid, {:event_duration, duration})
```

Comment:
At this moment that `duration` isn't persisted, it's just displayed on `metrics.duration.total` toolbar.